addons/account/models/account_move.py

```diff
@@ -1615,7 +1615,7 @@ def create_tax_cash_basis_entry(self, percentage_before_rec):
                                 'move_id': newly_created_move.id,
                                 'partner_id': line.partner_id.id,
                             })
-                            if line.account_id.reconcile:
+                            if line.account_id.reconcile and not line.reconciled:
```

Comment:
@hbto 

```suggestion
                            force = self.env['ir.config_parameter'].sudo().get_param(
            'force_unreconciled', True)
                            if line.account_id.reconcile and (force and not line.reconciled):
```

Como no es tan replicable el error, que te parece si en esta primer versión hacemos algo como esto. Por si se afecta alguna funcionalidad, simplemente se elimine el parametrro del sistema y el comportamiento sea el mismo que tiene Odoo ahora?

Nota: Mi propuesta de código es un ejemplo, hay que ver cual es la mejor solución.

Saludos